Symfony/CS/Fixer/PSR2/MultipleUseFixer.php

```diff
@@ -25,6 +25,14 @@ class MultipleUseFixer extends AbstractFixer
     /**
      * {@inheritdoc}
      */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound(T_USE);
```

Comment:
I don't think it would improve much (``,`` appears in function signatures and in function calls, so I doubt it will exclude much files)